acceptance/tests/facts/windows.rb

```diff
@@ -0,0 +1,95 @@
+test_name "Facts should resolve as expected in Windows 2012R2"
+
+#
+# This test is intended to ensure that facts specific to an OS configuration
+# resolve as expected in Windows 2012R2.
+#
+# Facts tested: os, processors, ruby, networking, identity, kernel
+#
+
+confine :to, :platform => "windows-2012"
```

Comment:
That being said, I'll put this back into Doing. Was mostly just putting it up for some early review since most of the platforms are present